tests/lang/eval-fail-hashfile-missing.nix

```diff
@@ -0,0 +1,5 @@
+let
+  paths = [ ./this-file-is-definitely-not-there-7392097 "/and/neither/is/this/37293620" ];
+in
+  builtins.concatLists (map (hash: map (builtins.hashFile hash) paths) ["md5" "sha1" "sha256" "sha512"])
```

Comment:
Having more than one test here is probably pointless since evaluation will fail on the first one, if I'm not mistaken.

Reply:
You're right. But if you run `make installcheck` the test will fail i.e. the evaluation  succeeds. But I think something else is wrong which makes the test fail.